test_dask_multiprocessing fails with pip packages

Any hints how to debug this?

[obs buildlog](https://github.com/spyder-ide/spyder-kernels/files/4904356/daskfail.log)
```
[    3s] + pytest-3.8 --ignore=_build.python2 --ignore=_build.python3 --ignore=_build.pypy3 -v -v
[    4s] ============================= test session starts ==============================
[    4s] platform linux -- Python 3.8.3, pytest-5.4.3, py-1.9.0, pluggy-0.13.1 -- /usr/bin/python3
[    4s] cachedir: .pytest_cache
[    4s] rootdir: /home/abuild/rpmbuild/BUILD/spyder-kernels-1.9.2
[    4s] plugins: flaky-3.6.1
[    5s] collecting ... collected 56 items
...
[   21s] =================================== FAILURES ===================================
[   21s] __________________________ test_dask_multiprocessing ___________________________
[   21s] 
[   21s] tmpdir = local('/tmp/pytest-of-abuild/pytest-31/test_dask_multiprocessing2')
[   21s] 
[   21s]     @flaky(max_runs=3)
[   21s]     @pytest.mark.skipif(not PY3,
[   21s]                         reason="Only meant for Python 3")
[   21s]     def test_dask_multiprocessing(tmpdir):
...
[   21s]             msg = client.get_shell_msg(block=True, timeout=TIMEOUT)
[   21s]             content = msg['content']
[   21s] >           assert content['found']
[   21s] E           assert False
[   21s] 
[   21s] spyder_kernels/console/tests/test_console_kernel.py:477: AssertionError
[   21s] =============================== warnings summary ===============================
...
[   21s] ===Flaky Test Report===
[   21s] 
[   21s] test_cwd_in_sys_path passed 1 out of the required 1 times. Success!
[   21s] test_multiprocessing passed 1 out of the required 1 times. Success!
[   21s] test_dask_multiprocessing failed (2 runs remaining out of 3).
[   21s] 	<class 'AssertionError'>
[   21s] 	assert False
[   21s] 	[<TracebackEntry /home/abuild/rpmbuild/BUILD/spyder-kernels-1.9.2/spyder_kernels/console/tests/test_console_kernel.py:477>]
[   21s] test_dask_multiprocessing failed (1 runs remaining out of 3).
[   21s] 	<class 'AssertionError'>
[   21s] 	assert False
[   21s] 	[<TracebackEntry /home/abuild/rpmbuild/BUILD/spyder-kernels-1.9.2/spyder_kernels/console/tests/test_console_kernel.py:477>]
[   21s] test_dask_multiprocessing failed; it passed 0 out of the required 1 times.
[   21s] 	<class 'AssertionError'>
[   21s] 	assert False
[   21s] 	[<TracebackEntry /home/abuild/rpmbuild/BUILD/spyder-kernels-1.9.2/spyder_kernels/console/tests/test_console_kernel.py:477>]
[   21s] test_runfile passed 1 out of the required 1 times. Success!
[   21s] test_np_threshold passed 1 out of the required 1 times. Success!
[   21s] test_matplotlib_inline passed 1 out of the required 1 times. Success!
[   21s] 
[   21s] ===End Flaky Test Report===
[   21s] =========================== short test summary info ============================
[   21s] FAILED spyder_kernels/console/tests/test_console_kernel.py::test_dask_multiprocessing
[   21s] ============ 1 failed, 53 passed, 2 skipped, 50 warnings in 17.49s =============
```

--- a/setup.py
+++ b/setup.py
@@ -47,7 +47,7 @@ REQUIREMENTS = [
 TEST_REQUIREMENTS = [
     'codecov',
     'cython',
-    'dask',
+    'dask[distributed]',
     'flaky',
     'matplotlib',
     'mock',

